docs/docs/developers/reference/environment_reference/cli_wallet_reference.md

```diff
@@ -122,13 +150,74 @@ This will mint the max amount of fee juice on L1 and bridge it to L2.
 aztec-wallet bridge-fee-juice <AztecAddress>
 ```
 
+## Connect to the Testnet
+
+To connect to the testnet, pass the `AZTEC_NODE_URL` to the wallet with the `--node-url` (`-n`) option.
+
+```bash
+export AZTEC_NODE_URL=<testnet-ip-address>
+# Get the canonical sponsored FPC address to pay tx fees
+aztec get-canonical-sponsored-fpc-address
```

Comment:
Thoughts on just hardcoding this in our docs?

Reply:
fine with me, will add